### PR TITLE
yurt-app-manager install by chart

### PR DESCRIPTION
**Pull request by huiwq1990**
Signed-off-by: huiwq1990 <huiwq1990@163.com>

### CODE REVIEW

File: docs/installation/manually-setup.md [hunk 1]
```diff
@@ -75,14 +75,22 @@ Then edit the config/setup/yurt-controller-manager.yaml
...
+
+```shell
+helm repo add openyurt https://openyurtio.github.io/openyurt-helm
 ```
-$ kubectl apply -f config/setup/all_in_one.yaml
+
+Install `yurt-app-manager` use command as follows:
+
+```shell
+helm upgrade --crate-namespace openyurt --install yurt-app-manager openyurt/yurt-app-manager
```
Comment (rambohe-ch) [inline docs/installation/manually-setup.md:87]: typo error: 
--crate-namespace openyurt --> --create-namespace kube-system

File: docs/installation/manually-setup.md [hunk 2]
```diff
@@ -75,14 +75,22 @@ Then edit the config/setup/yurt-controller-manager.yaml
...
 ```
 
-Wait for the yurt-app-manager operator to be created successfully
+Check `yurt-app-manager` pod if startup:
+
 ```
-$ kubectl get pod -n kube-system | grep yurt-app-manager
+kubectl get pod -n openyurt | grep yurt-app-manager
```
Comment (rambohe-ch) [inline docs/installation/manually-setup.md:93]: maybe it's better to install `yurt-app-manager` in `kube-system` namespace

File: i18n/zh/docusaurus-plugin-content-docs/current/installation/manually-setup.md [hunk 3]
```diff
@@ -72,14 +72,22 @@ $ kubectl create secret docker-registry dockerpass --docker-username=your-docker
...
 ```
-$ kubectl apply -f config/setup/all_in_one.yaml
+helm repo add openyurt https://openyurtio.github.io/openyurt-helm
 ```
 
-确认yurt-app-manager组件已经成功创建:
+安装`yurt-app-manager`:
+
+```shell
+helm upgrade --crate-namespace openyurt --install yurt-app-manager openyurt/yurt-app-manager
```
Comment (rambohe-ch) [inline i18n/zh/docusaurus-plugin-content-docs/current/installation/manually-setup.md:84]: ddiot

File: docs/installation/manually-setup.md (hunk 2 above)
Comment (huiwq1990) [reply docs/installation/manually-setup.md:93]: @rambohe-ch OpenYurt's component should not share namespace with kube-system. We should treat as a standalone component.

Comment (rambohe-ch) [reply docs/installation/manually-setup.md:93]: @huiwq1990 I think we should deploy OpenYurt components in kube-system namespace on default mode, and create a new namespace named openyurt. meanwhile, end user can deploy OpenYurt components in other namespace if they want.

**Comment by huiwq1990** [comment]
@rambohe-ch

**Review by rambohe-ch — APPROVED** [review]

**Pull request merged**